packages/block-editor/src/utils/transform-styles/test/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import traverse from '../traverse';
+import parse from '../ast/parse';
+
+describe( 'CSS traverse', () => {
+	it( 'Should traverse the CSS', () => {
+		const input = `h1 { color: red; }`;
+		const output = traverse( input, ( node ) => {
+			if ( node.type === 'rule' ) {
+				return {
+					...node,
+					selectors: node.selectors.map(
+						( selector ) => 'namespace ' + selector
+					),
+				};
+			}
+
+			return node;
+		} );
+
+		expect( output ).toMatchSnapshot();
+	} );
+} );
+
+describe( 'CSS parse', () => {
+	it( 'Should handle closing parenthesis within url', () => {
+		const input = `.a { b: url(")"); d: url(";a"); }`;
```

Comment:
String interpolation isn't necessary.

```suggestion
		const input = '.a { b: url(")"); d: url(";a"); }';
```